apps/designer/app/shared/feature-flags/feature.ts

```diff
@@ -28,5 +28,5 @@ export const isFeatureEnabled = (name: Name): boolean => {
   const defaultValue = flags[name];
   const envValue = parse(env.FEATURE).includes(name);
   const localValue = readLocal().includes(name);
-  return localValue ?? envValue ?? defaultValue;
+  return localValue || envValue || defaultValue;
```

Comment:
What is the difference?

Reply:
includes `''`

Reply:
Ouch. I wonder if there is a less ambiguous syntax or a helper function we could use for such cases.

Reply:
Because it's confusing even to me (an experienced dev) :)

Reply:
the reason I supported empty string is just because some may set that flag to en empty string in local storage to disable feature, its not really important, just thought would be more practical

Reply:
I know remembered why I added those, its not because of empty string, its because of `false`
we return false from .includes, which makes the first value always take precedence with ??

Reply:
I get the logic but I think the amount of gotcha with `||` and `??` what makes me a bit uncomfortable reading it.